deploy/helm/node-feature-discovery-operator/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for node-feature-discovery-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+namespace: node-feature-discovery-operator
+image:
+  repository: gcr.io/k8s-staging-nfd/node-feature-discovery-operator
+  # This should be set to 'IfNotPresent' for released version
+  pullPolicy: Always
+  # tag, if defined will use the given image tag, else Chart.AppVersion will be used
+  # tag
+imagePullSecrets: []
+
+nameOverride: ""
+fullnameOverride: ""
+namespaceOverride: ""
+
+enableNodeFeatureApi: false
+
+nfd:
+  image: 
+    repository: k8s.gcr.io/nfd/node-feature-discovery
```

Comment:
Hmm, wondering should we point to nfd devel branch as well? That'd make it easier to test the devel version of operator against devel version of the operand. WDYT?

Reply:
Usually no, the operator as of today, has been in "reactive" mode against the Operand (NFD), it waits for new features and then enables then, if we set Devel:Devel as you say, a lot of the times that could result in a broken deployment.

Although....  we DO NOT recommend deploying from master into prod, that's why we cut stable releases, so..... maybe yes, I am up for it.

Reply:
K, I saw you changed this. I don't have strong opinions. Probably then the policy should be that when creating a release branch you'd fix this into some (latest) released operand version